circle.yml

```diff
@@ -24,6 +25,7 @@ test:
     - env CXXFLAGS="-Werror" make test
     - valgrind --leak-check=full --show-leak-kinds=definite,indirect,possible --error-exitcode=1 ./bin/test-libdrafter 2> memcheck.log
     - valgrind --leak-check=full --show-leak-kinds=definite,indirect,possible --error-exitcode=1 ./bin/test-capi 2> memcheck-capi.log
+    - ./tools/clang-format-check.sh
```

Comment:
I'd rather have clang-format reject my changes before the build, not after. It would be frustrating to wait for a successful build just to get an error message about formatting at the end.